ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/TraceCall.java

```diff
@@ -119,34 +103,8 @@ protected Object resultByBlockNumber(
 
     final Block block = blockchainQueries.get().getBlockchain().getChainHeadBlock();
 
-    final TraceCallResult.Builder builder = TraceCallResult.builder();
```

Comment:
cc @pinges  we might wanna wait to see this gets merged before we do our implementation?